py/desispec/scripts/preproc.py

```diff
@@ -79,8 +79,9 @@ def parse(options=None):
                         help = 'specify a difference ccd calibration filename (for dev. purpose), default is in desispec/data/ccd')
     parser.add_argument('--fill-header', type = str, default = None,  nargs ='*', help="fill camera header with contents of those of other hdus")
     parser.add_argument('--scattered-light', action="store_true", help="fit and remove scattered light")
-    parser.add_argument('--psf', type = str, required=False, default=None, help="psf file to remove scattered light")
-    
+    parser.add_argument('--psf', type = str, required=False, default=None, help="psf file to remove scattered light or to compute the variance model")
+    parser.add_argument('--variance-model', action="store_true", help="compute a model of the CCD image to derive the Poisson noise")
```

Comment:
Minor: See earlier comment about consider converting this to a verb `--model-variance`.

Reply:
done